synadm/cli/notice.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# synadm
+# Copyright (C) 2020-2022 Philip (a-0)
+#
+# synadm is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# synadm is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""server notice-related CLI commands
+"""
+
+import click
+from click_option_group import optgroup, MutuallyExclusiveOptionGroup
+from click_option_group import RequiredAnyOptionGroup
+
+from synadm import cli
+
+@cli.root.group()
+def notice():
+    """ Send server notices to local users
+    """
+
+@notice.command(name="send")
+@click.option("--from-file/--from-argument", "-f/-a", default=False,
+    help="Interpret arguments as file paths instead of notice content and read content from those files")
+@click.option("--batch", "--non-interactive", is_flag=True, default=False,
```

Comment:
Remove this option and use the global option --batch. you can access its value from helper.batch attribute.